### PR TITLE
new modules

### PR DESCRIPTION
**Pull request by eredi93**
* add load_balancers variable to ASG module
* new ELB module
* new IAM user and role module
* script that creates the template for a new module

### CODE REVIEW

File: elb/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## ELB
+
+Module usage:
+
+```
+module "elb" {
+  source = "github.com/intercom/terraform-modules//elb"
+
+  fleet_name          = "${var.fleet_name}"
```
Comment (austinhalpin) [inline elb/README.md]: Are we going to have this talk again? 😅

Comment (eredi93) [inline elb/README.md]: yes in this case it make sense as an ELB is tight to an ASG and not an whole app

Comment (austinhalpin) [inline elb/README.md]: I think `endpoint_name` would be better. You can attach more than 1 fleet to an ASG.

Comment (eredi93) [inline elb/README.md]: where the security groups very at the app level an LB is tight to a fleet.
a fleet name would be `<app-name>-<env>-<cluster>` eg `my_cool_app-production-web` if i take the same approch of the security groups i will end up with a naming conflict as an app can have multiple fleet and so several LB

**Comment by eredi93** [comment]
hahaha that bash script helps me out creating a template for a new module. time saving is always good 🚀

**Pull request merged**